packages/authentication/src/utils/constants.js

```diff
@@ -0,0 +1,19 @@
+// External dependencies
+const { ExtractJwt } = require('passport-jwt');
+// Internal dependencies
+const { jwtParams } = require('../../config');
+
+const jwtOptions = {
+  jwtSecret: jwtParams.secret,
+  jwtFromRequest: ExtractJwt.fromAuthHeaderAsBearerToken,
+};
+
+const tokenExpirationTime = {
+  machine: Math.floor(Date.now / 1000) + (60 * 60 * 24),
+  user: Math.floor(Date.now / 1000) + (60 * 60),
+};
```

Comment:
You should manage this (60*60*24) as a constant to understand if you are talking about time

Reply:
Well I can do this due to how nodejs resolve their dependencies and events. In each request node generate a new event, new dependencies and new instance for their constants. So even though they are constants is going to change in every request